src/Features/CSharp/Portable/ConvertNamespace/ConvertNamespaceDiagnosticAnalyzer.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.CodeStyle;
+using Microsoft.CodeAnalysis.CSharp.CodeStyle;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.CSharp.ConvertNamespace
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    internal class ConvertNamespaceDiagnosticAnalyzer : AbstractBuiltInCodeStyleDiagnosticAnalyzer
```

Comment:
Is there a specific reason why the analyzer is being added to Features layer as opposed to the shared [Analyzers](https://github.com/dotnet/roslyn/tree/main/src/Analyzers) layer. Doing the latter would automatically add it to the CodeStyle NuGet package that gets inserted into the .NET SDK and enable it in CI when user bumps the rule to a warning/error.

Reply:
nope.  i can put it in there.  i'm just used to doing it this way :)